pegasus/sites.v3/code.org/views/share_privacy.haml

```diff
@@ -0,0 +1,66 @@
+%div{style: 'position:absolute; top:5px; right:5px', id: 'share_privacy_policy'}
+  -# haml-lint:disable MultilinePipe
+  - email_body = "Dear parents,\n\n" + |
+                  "Software and technology are everywhere - in our computers, in our pockets, in our cars, " + |
+                  "and used in banking, communications, entertainment, transportation, and science, to name a few. " + |
+                  "However, very few kids are learning how to actually create technology. In today’s world, " + |
+                  "computer science is fundamental for preparing students to be informed and successful in our " + |
+                  "increasingly technological society, regardless of what career path their future holds.\n\n" + |
+                  "This year we’re proud to be teaching computer science in our classroom using Code.org’s online " + |
+                  "learning platform and curriculum. Code.org takes student privacy very seriously and has " + |
+                  "earned one of the highest privacy ratings from Common Sense Media.  You can read their " + |
+                  "privacy notice here: https://code.org/privacy/student-privacy\n\n" + |
+                  "I’d encourage you to ask your student about what they are learning and have them show " + |
+                  "you the programs or games they create this year!\n\n" + |
+                  "Thanks," |
```

Comment:
nit: pipe needed at end?

Reply:
Yes, for some reason. http://haml.info/docs/yardoc/file.REFERENCE.html#multiline